liferay-echo/liferay-portal
# LPD-19079 Implement the copy action for folders and multiple elements

> [!NOTE]
>You need the FF  LPS-189856. This pr contains also the commits from: https://github.com/liferay-echo/liferay-portal/pull/15376 

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-19079)

### Changes

Add the bulk action to move folders and display page templates.

## Test scenario 

1. Create a DPT
2. Create some folders 
3. Check the copy action for each one individually using the dropdown
4. Check the copy action in bulk slecting different elements

https://github.com/liferay-echo/liferay-portal/assets/93203423/e810a3c9-7573-4bdd-89c2-b17d598c8690

https://github.com/liferay-echo/liferay-portal/assets/93203423/1875e1a2-c7ba-4200-8494-825c5edceaaf

https://github.com/liferay-echo/liferay-portal/assets/93203423/8e8ddccb-1245-43f8-b944-f01e32f2f5c9
